2-mandatory/2-financial-times.js

```diff
@@ -4,34 +4,71 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
-function potentialHeadlines(allArticleTitles) {
-    // TODO
+function potentialHeadlines(allArticleTitles){
+    let articlelCharacters = [];
+        for(let title of allArticleTitles){
+        if(title.length <= 65){
+        articlelCharacters.push(title);
+    }
+    }
+        return articlelCharacters;
 }
-
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let shortestHeadline;
+    let fewestNumberOfWords;
+
+    for(let title of allArticleTitles) {
+        // find number of words in title
+        let numberOfWords = title.split(' ').length;
+        
+        // compare the number with fewestNumberOfWords
+        if(fewestNumberOfWords === undefined || numberOfWords < fewestNumberOfWords) {
+            // if it's less, update fewestNumberOfWords and shortestHeadline
+            fewestNumberOfWords = numberOfWords;
+            shortestHeadline = title;
+        }
+        // if it's not less, do nothing
+    }
+
+    return shortestHeadline;
 }
 
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
```

Comment:
This looks very good!
I like that you used a separate function here - this is a good example of how to break a problem down into smaller, simpler problems 👍 
One small note: when you write code like this - `if (containNumbers(title)== true){`, it can usually be re-written to `if (containNumbers(title)){`. Can you think of what that is?